homeassistant/components/bsblan/strings.json

```diff
@@ -24,5 +24,13 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     }
+  },
+  "exceptions": {
+    "set_preset_mode_error": {
+      "message": "Can't set preset mode to {preset_mode} when HVAC mode is not set to auto"
+    },
+    "set_data_error": {
+      "message": "An error occurred while sending the data to the BSBLAN device"
```

Comment:
Would it be beneficial to have the exception message in here?

Reply:
Hm good question, but the problem would be that the message is translated and the actual exception message is not. HomeWizard for example don't include the exception message as well.

Reply:
As it's `raise .. from err` would the message appear in the logs? I guess that would be sufficient then.

Reply:
Yes, the full stack trace is in the log.